allauth_ui/templates/account/password_reset_key_message.html

```diff
@@ -0,0 +1,60 @@
+{% load i18n %}{% autoescape off %}
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            line-height: 1.6;
+            color: #333;
+            max-width: 600px;
+            margin: 0 auto;
+            padding: 20px;
+        }
+        .button {
+            display: inline-block;
+            padding: 10px 20px;
+            background-color: #7f22fe;
+            color: white !important;
+            text-decoration: none;
+            border-radius: 5px;
+            margin: 20px 0;
+        }
+        .footer {
+            margin-top: 30px;
+            padding-top: 20px;
+            border-top: 1px solid #eee;
+            font-size: 12px;
+            color: #666;
+        }
+    </style>
+</head>
+<body>
+    <h2>{% blocktrans with site_name=current_site.name %}Hello from {{ site_name }}!{% endblocktrans %}</h2>
+
+    {% if user.first_name %}
+    <p>Dear {{ user.first_name }},</p>
+    {% else %}
+    <p>Hello,</p>
+    {% endif %}
+
+    <p>{% blocktrans with site_name=current_site.name %}You are receiving this email because a password reset was requested for your {{ site_name }} account. If you did not make this request, please ignore this email.{% endblocktrans %}</p>
+
+    <p>{% blocktrans %}To reset your password, please click the button below:{% endblocktrans %}</p>
+
+    <a href="{{ password_reset_url }}" class="button">Reset Password</a>
+
+    <p>{% blocktrans %}This password reset link will expire in a few hours for security reasons.{% endblocktrans %}</p>
+
+    <p>{% blocktrans %}If you have any issues, please contact our support team.</p>
+
+    <div class="footer">
+        {% blocktrans with site_name=current_site.name site_domain=current_site.domain %}
+        Thank you for using {{ site_name }}!<br>
+        <a href="https://{{ site_domain }}">{{ site_domain }}</a>
+        {% endblocktrans %}
```

Comment:
Looking at the source, I don't think site_domain is available in the context :/

https://github.com/pennersr/django-allauth/blob/main/allauth/templates/account/email/password_reset_key_message.txt

I think this is the function that sends the email. Take a look at the context:

https://github.com/pennersr/django-allauth/blob/434c4c073642980b6777eab9b67f1161d6c2f522/allauth/account/internal/flows/password_reset.py#L93-L122

Reply:
I believe it gets the site_domain from the sites framework. It worked when I tried it on the backend repo so believe it should work more generally here.